runelite-client/src/main/java/net/runelite/client/plugins/itemcharges/ItemChargeConfig.java

```diff
@@ -47,6 +47,7 @@
 	String KEY_EXPLORERS_RING = "explorerRing";
 	String KEY_RING_OF_FORGING = "ringOfForging";
 	String KEY_BLOOD_ESSENCE = "bloodEssence";
+	String KEY_SKULL_SCEPTRE_I = "skullSceptre(i)";
```

Comment:
I don't know if () is safe here, it is probably best to not include them

Reply:
Removed ()